dockers/docker-base/Dockerfile.j2

```diff
@@ -52,6 +52,9 @@ COPY ["etc/rsyslog.conf", "/etc/rsyslog.conf"]
 COPY ["etc/rsyslog.d/*", "/etc/rsyslog.d/"]
 COPY ["root/.vimrc", "/root/.vimrc"]
 
+RUN pip install --upgrade 'pip<21'
+RUN apt-get purge -y python-pip
```

Comment:
not sure why to touch it.

Reply:
This is simply to align the way we install/upgraded pip everywhere